content/calendar/CSSC.csv

```diff
@@ -0,0 +1,3 @@
+name, description, start_date, start_time, end_date, end_time
```

Comment:
I thought we're getting these events from another repo. If these files are duplicated, let's remove them